geoportailv3/static/js/maincontroller.js

```diff
@@ -82,6 +84,15 @@ app.MainController.prototype.setMap_ = function() {
 
   /** @type {ol.Map} */
   this['map'] = new ol.Map({
+    controls: [
+      new ol.control.Zoom({zoomInLabel: '', zoomOutLabel: ''}),
+      new ol.control.ZoomToExtent(
+          {extent:
+                [425152.9429259216, 6324465.99999133,
+                 914349.9239510496, 6507914.867875754]
+          }),
```

Comment:
These extent coordinates should go into a global value or constant if reused elsewhere.
We don't need it now though.